packages/editor/src/components/blocks/layout/Layout.tsx

```diff
@@ -67,9 +68,9 @@ export const LayoutComponent: ComponentConfig<LayoutProps> = {
     type: { subsection: 'General', label: 'Type', type: 'select', options: typeOptions },
     justifyContent: {
       subsection: 'General',
-      type: 'select',
+      type: 'generic',
```

Comment:
maybe it would be nice if we register this toggle group as known field type and not just use the generic way for very specific things